src/execution/compiler/compiler.cpp

```diff
@@ -98,6 +100,8 @@ void Compiler::Run(Compiler::Callbacks *callbacks) {
 
   callbacks->EndPhase(Phase::BytecodeGeneration, this);
 
+  ast::AstPrettyPrint::Dump(std::cerr, this->GetAST());  // NOLINT
```

Comment:
make sure this gets removed before merge

Reply:
the include above too

Reply:
done